main.py

```diff
@@ -337,6 +342,50 @@ def isPaused(self):
 	def isRunning(self):
 		return self._p is not None # FIXME: Check subproc state
 
+class AppVeyor(object):
+	def __init__(self, repo, branch, user):
+		self.gjson = json.loads(requests.get("https://ci.appveyor.com/api/projects/" + user + "/" + repo + "/branch/" + branch).text)
+
+	def GetJobs(self):
+		return self.gjson["build"]["jobs"]
+
+	def GetRelease(self, id):
+		idjson = json.loads(requests.get("https://ci.appveyor.com/api/buildjobs/" + str(id) + "/artifacts").text)
+		return idjson
+
+class Updater(object):
+    def __init__(self, settings):
+        self.gsettings = settings
```

Comment:
What does `g` in `gsettings` stand for?

Reply:
i don't remember exactly, pretty sure for global, i will rename those variable in the next commit.